docs/code-organization/git-quickstart.mdx

```diff
@@ -0,0 +1,331 @@
+remote: Total 204 (delta 104), reused 181 (delta 91), pack-reused 0
+Receiving objects: 100% (204/204), 72.83 KiB | 298.00 KiB/s, done.
+Resolving deltas: 100% (104/104), completed with 25 local objects.
+From github.com/ntuscse/docs
+[More output omitted]
+```
+
+After checking your code still works or after you finish fixing it, it's time
+to show it to everyone by pushing it to GitHub with [`git push`][git-push]!
+
+```
+$ git push
+Total 204 (delta 104), reused 181 (delta 91), pack-reused 0
+remote:
+remote: Create a pull request for 'docs/SCSE-24' on GitHub by visiting:
+remote:      https://github.com/ntuscse/docs/pull/new/docs/SCSE-24
+remote:
+To github.com/ntuscse/docs
+ * [new branch]      docs/SCSE-24 -> docs/SCSE-24
+```
+
+If you forgot to create a branch, you can still specify what the name at GitHub
+should be. In the following example, we assume you worked on `main` and the
+correct name should be `docs/SCSE-24`.
+
+```
+$ git push origin main:docs/SCSE-24
+Total 204 (delta 104), reused 181 (delta 91), pack-reused 0
+remote:
+remote: Create a pull request for 'docs/SCSE-24' on GitHub by visiting:
+remote:      https://github.com/ntuscse/docs/pull/new/docs/SCSE-24
+remote:
+To github.com/ntuscse/docs
+ * [new branch]      main -> docs/SCSE-24
+```
+
+## What Next?
+
+This document is currently work in progress! If there is anything you are
+unclear about, ask about them in our chat and we will be using those valuable
+feedbacks to improve this page.
+
+Git is just a tool to help manage different versions of code as we collaborate
+and is nothing if we don't use it so start familiarizing yourself with it! You
+may wish to do so by [creating your own GitHub repository][create-github-repo]
+and playing around with it.
+
+You might want to check out the [naming conventions](naming-convention) we have
+written to help keep everything tidy.
+
+## Appendix A: Fixing History
+
+So you did not heed my advice in [mistake fixing](#fixing-mistakes) and
+recreated history that has already been pushed. Now, GitHub is angry at you
+whenever you try to push.
+
+```
+$ git push
+To github.com/ntuscse/docs
+ ! [rejected]        docs/SCSE-24 -> docs/SCSE-24 (non-fast-forward)
+error: failed to push some refs to 'github.com/ntuscse/docs'
+hint: Updates were rejected because the tip of your current branch is behind
+hint: its remote counterpart. Integrate the remote changes (e.g.
+hint: 'git pull ...') before pushing again.
+hint: See the 'Note about fast-forwards' in 'git push --help' for details.
+```
+
+You have three options, all of them pretty nuclear but that is the state we are
+in.
+
+#### Option 1: Force push (softer version)
+
+Perhaps you were just pushing to the branch of your PR and it's actually fine
+to override it.
+
+:::caution
+Before we continue, make extra sure that you are pushing to the branch of your
+PR and not someone else's branch. You can check this by reading the right side
```

Comment:
reword to "make sure you are pushing the branch of your own task, and not someone elses.....", or something to that effect.